src/components/screen-renderer.vue

```diff
@@ -53,6 +75,11 @@ export default {
     },
   },
   methods: {
+    hash(s) {
+      for (var i=0,h=9;i<s.length;)
+        h=Math.imul(h^s.charCodeAt(i++),9**9);
+      return h^h>>>9;
```

Comment:
this is invalid javascript code, line 81, the h is not being picked as you think from the var declared in line 79
`h` is in the context of the for loop, not in the context of the function itself

Reply:
As in C i, h variables are outside the for block scope. At line 81, i has the string length, for example